suite-native/module-onboarding-legacy/src/screens/WelcomeScreen.tsx

```diff
@@ -56,6 +57,7 @@ export const WelcomeScreen = () => {
             <Box flex={1} />
             <Box alignItems="center" justifyContent="center">
                 <Box alignItems="center">
+                    <E2ESkipOnboardingButton />
```

Comment:
Not sure this change makes sense, this screen will be removed pretty soon. @PeKne may provide further details.

Reply:
This change currently allows to skip onboarding in E2Es. I expect that once the new onboarding is introduced, we will add `E2ESkipOnboardingButton` component there as well.